.github/workflows/build.yml

```diff
@@ -0,0 +1,68 @@
+name: "Build the images, also push them if merged on main"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      max-parallel: 3
+      matrix:
+        target:
+#          - { directory: rhel, label: ":linux: Glibc build images" }
+#          - { directory: musl, label: ":linux: Musl (dynamic) build images" }
+          - { directory: linux-static, label: ":linux: Musl (static) build images" }
```

Comment:
On Buildkite `:linux:` would turn into a logo but it doesn’t work like this on GH Actions so we probably want just `Linux`. I penguin emoji would also work I guess.

Reply:
same for all other logos